config/dev.exs

```diff
@@ -26,5 +26,3 @@ config :phoenix, :stacktrace_depth, 20
 
 # Initialize plugs at runtime for faster development compilation
 config :phoenix, :plug_init_mode, :runtime
-
-config :logger, level: :info
```

Comment:
Why? Do we want to include `debug` logs by default as well?

Reply:
Now, log level is configured in runtime.exs where info is the default value

Reply:
Ah yes my mistake, thought it was in prod not runtime 😅